app/javascript/designSystem/__stories__/typography.scss

```diff
@@ -0,0 +1,6 @@
+p::after {
```

Comment:
This just shows in the typography examples the classes used to generate that particular piece of text, e.g.

![image](https://user-images.githubusercontent.com/833231/76558128-213ec580-6473-11ea-815f-e7ef4066f159.png)

I thought it was a good idea, but welcome to feedback @ludwiczakpawel. Also, I just randomly picked orange for the color.

Reply:
sooo soo good! <3